### PR TITLE
Add CHANGELOG for 1.10.2

### PR DESCRIPTION
**Pull request by MarcosDY**
Add CHANGELOG for 1.10.2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+### Changed
+
+- Delegated Identity API to allow subscription by process ID (#5272)
+- Agent Debug endpoint to count SVIDs by type (#5352)
+- Agent health check to report an unhealthy status until Agent SVID is secured (#5298)
+- Small documentation improvements (#5393)
+
+### Fixed
+
+- `aws_iid` NodeAttestor to properly handle network interfaces (#5300)
```
Comment (amartinezfayo) [inline CHANGELOG.md:20]:
```suggestion
- `aws_iid` NodeAttestor to properly handle multiple network interfaces (#5300)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+- Server configuration to correctly propagate `sql_transaction_timeout` (#5345)
```
Comment (amartinezfayo) [inline CHANGELOG.md:21]:
```suggestion
- Server configuration to correctly propagate the `sql_transaction_timeout` setting in the experimental events-based cache (#5345)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [1.10.2] - 2024-09-03
+
+### Added
+
+- `http_challenge` NodeAttestor plugin (#4909)
+- Experimental support for validating container image signatures through Sigstore selectors in the docker Workload Attestor (#5272)
+- Metrics to track event-based cache (#5411)
```
Comment (amartinezfayo) [inline CHANGELOG.md:9]:
```suggestion
- Metrics that help monitoring the event-based cache (#5411)
```

**Review by amartinezfayo — COMMENTED** [review]
Thank you @MarcosDY! I have some suggestions.

File: CHANGELOG.md (hunk 3 above)
Comment (azdagron) [reply CHANGELOG.md:9]: Maybe "Metrics that help monitor the" or "Metrics for monitoring the"?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,25 @@
...
+
+### Changed
+
+- Delegated Identity API to allow subscription by process ID (#5272)
+- Agent Debug endpoint to count SVIDs by type (#5352)
+- Agent health check to report an unhealthy status until Agent SVID is secured (#5298)
```
Comment (azdagron) [inline CHANGELOG.md:15]:
```suggestion
- Agent health check to report an unhealthy status until the Agent is attested (#5298)
```

**Review by azdagron — APPROVED** [review]

**Review by amartinezfayo — APPROVED** [review]

**Pull request merged**